docs/source/hardware/lowpriorityFAQs.rst

```diff
@@ -0,0 +1,7 @@
+.. _partition_names:
+
+**********
+Low priority partition FAQs
+**********
+
+Or the FAQs can go here as another item in the main menu 
```

Comment:
Yes, as another item in the main menu is better, so as a new page.

Reply:
Done